packages/synthchron-simulator/src/simulation.ts

```diff
@@ -255,6 +274,4 @@ const checkTermination = <SpecificProcessModel extends ProcessModel, StateType>(
 const isEndOnAcceptingState = (
```

Comment:
The function name seems a bit too specific for such a general thing, but maybe it's fine

Reply:
Good point, I guess it's fine but we can rename at some point